src/nvim/keymap.c

```diff
@@ -318,73 +316,73 @@ static struct mousetable {
   {0,                         0,              0,      0},
 };
 
-/*
- * Return the modifier mask bit (MOD_MASK_*) which corresponds to the given
- * modifier name ('S' for Shift, 'C' for Ctrl etc).
- */
+/// Return the modifier mask bit (#MOD_MASK_*) corresponding to mod name
+///
+/// E.g. 'S' for shift, 'C' for ctrl.
 int name_to_mod_mask(int c)
+  FUNC_ATTR_CONST FUNC_ATTR_WARN_UNUSED_RESULT
```

Comment:
I thought "const" func attr cannot be used if globals (`mod_mask_table `) are referenced?

Reply:
It is immutable global.